src/status_im/contexts/wallet/sheets/network_selection/view.cljs

```diff
@@ -0,0 +1,76 @@
+(ns status-im.contexts.wallet.sheets.network-selection.view
+  (:require [quo.core :as quo]
+            [quo.foundations.resources :as quo.resources]
+            [react-native.core :as rn]
+            [status-im.contexts.wallet.common.utils :as utils]
+            [status-im.contexts.wallet.sheets.network-selection.style :as style]
+            [utils.i18n :as i18n]
+            [utils.money :as money]
+            [utils.re-frame :as rf]))
+
+(defn- network-item
+  [{:keys [network token on-select-network currency currency-symbol]}]
+  (let [{network-name :network-name
+         chain-id     :chain-id}                 network
+        {balances-per-chain :balances-per-chain} token
+        balance-for-chain                        (get balances-per-chain chain-id)
+        total-balance                            (money/bignumber (:balance balance-for-chain))
```

Comment:
Not sure, I believe we need to use `raw-balance` here.

But, If the balance is used, we can transform it to `bignumber` in

https://github.com/status-im/status-mobile/blob/d43e8881ef768629fb57c590ae30f672967410dc/src/status_im/contexts/wallet/data_store.cljs#L68-L72

this would help preventing conversion every time

cc: @ulisesmac

Reply:
@smohamedjavid / @ulisesmac I have updated the code use `raw-balance` instead. Please have another look and let me know if there is anything else I can improve.